lazy.config.ts

```diff
@@ -80,7 +80,7 @@ const config = {
 		},
 		'refresh-assets': {
 			execution: 'top-level',
-			baseCommand: `tsx <rootDir>/scripts/refresh-assets.ts`,
+			baseCommand: `tsx <rootDir>/scripts/refresh-assets.ts && biome format . --write`,
```

Comment:
why do we need this here? I think this could affect lazy's caching as it might touch files outside of what lazy expects